tests/resolvers/Mutation/assignToUserTags.spec.ts

```
@@ -0,0 +1,318 @@
+  expectedError,
+}: {
+  args: MutationAssignToUserTagsArgs;
+  context: { userId: string };
+  expectedError: string;
+}): Promise<void> => {
+  const { requestContext } = await import("../../../src/libraries");
+  const spy = vi
+    .spyOn(requestContext, "translate")
+    .mockImplementationOnce((message) => `Translated ${message}`);
+  try {
+    const { assignToUserTags: assignToUserTags } = await import(
```

Biome

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)